src/components/Graphs/Graphs.tsx

```
@@ -0,0 +1,111 @@
+
+const CustomizedLabel: FunctionComponent<any> = (props: any) => {
+  const { x, y, stroke, value } = props;
+
+  return (
+    <text x={x} y={y} dy={-4} fill={stroke} fontSize={10} textAnchor="middle">
+      {value}
+    </text>
+  );
+};
+
+const CustomizedAxisTick: FunctionComponent<any> = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)